pfcpiface/go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/Showmax/go-fqdn v1.0.0
 	github.com/golang/protobuf v1.4.3
 	github.com/prometheus/client_golang v1.8.0
-	github.com/wmnsk/go-pfcp v0.0.8-0.20201022125237-73401f105edc
```

Comment:
is this automatically updated or we need to update it manually?

Reply:
Because we wanted a particular commit that was not yet tagged at that time, we updated it manually by running `go get <>@master`. Now that its been tagged, we ran it again without `@master` with which the tag 0.0.8 is picked up.

In regular cases I have enabled dependabot to open pull request whenever our dependency is updated.